libnavigation-core/src/main/java/com/mapbox/navigation/core/formatter/MapboxDistanceUtil.kt

```diff
@@ -105,132 +268,44 @@ object MapboxDistanceUtil {
      * @param unitType indicates whether the value should be returned as metric or imperial
      * @return an object containing values for displaying the formatted distance
      */
+    @JvmOverloads
```

Comment:
wdyt if change the signature to wrapper (builder included) to support future possible changes?

Reply:
Do you mean introduce a new method like
`formatDistance(Wrapper)`
where `Wrapper` is a model class with builder?
I'm not sure. I mean if we add a new parameter we will probably need it in our calculations. Which means that we will have to provide the default value anyway to be backward compatible. How is Builder going to help here?

Reply:
java developers cannot fully rely on default parameters. For instance in the method 
```kotlin
@JvmOverloads
fun method(
  a: A,
  b: B = defaultB,
  c: C = defaultC
)
```

java developers cannot overload the default value for param `b`, and keep the same for `c`, so invoking of method on the java side would look like (we're forcing to set param `c`, even if default value exists)

```java
#method(a, customValuesB, defaultC)
```

Reply:
I think you meant the other way around.
But OK, agreed that default parameters may be tricky to support in the future so I'll add a new method just the way it's done for the `formatDistance` that returns `FormattedDistanceData`.